App/Talent.App.WebApp/wwwroot/js/react/ReactScripts/Employer/ManageJob/JobSummaryCard.jsx

```diff
@@ -1,19 +1,126 @@
 ﻿import React from 'react';
 import Cookies from 'js-cookie';
-import { Popup } from 'semantic-ui-react';
+import { CardMeta,
+    CardHeader,
+    CardGroup,
+    CardDescription,
+    CardContent,
+    Button,
+    Card,
+    Icon,
+    Popup } from 'semantic-ui-react';
 import moment from 'moment';
+import { Link } from 'react-router-dom';
 
 export class JobSummaryCard extends React.Component {
     constructor(props) {
         super(props);
-        this.selectJob = this.selectJob.bind(this)
+        this.selectJob = this.selectJob.bind(this);
+
+        this.state = {
+            jobs: this.props.jobData,
+            cards: []
+        };
+    }
+
+    convertLocation(location) {
+        let locationMeta = '';
+        if (location) {
+            let { city, country } = location;
+            if (city && country) {
+                locationMeta = `${city}, ${country}`;
+            } else if (city) {
+                locationMeta = city;
+            } else if (country) {
+                locationMeta = country;
+            }
+        }
+
```

Comment:
This looks good but seems too long. Can you try to implement something like this
`if (!location) return '';
    const { city, country } = location;
   locationMeta = city && country ? `${city}, ${country}` : city || country || '';`
   return locationMeta